code/modules/cciaa/cciaa_items.dm

```diff
@@ -1,5 +1,9 @@
 //CCIAA's tape recorder
 /obj/item/device/taperecorder/cciaa
+	name = "Human Resources Recorder"
+	desc = "A recorder modified for the specific use-cases of human resource departments around the galaxy."
```

Comment:
```suggestion
	desc = "A modified recorder used for recording interviews by human resources personnel around the galaxy."
```

Reply:
I dont like the "recorder used for recording" part, its redundant.